mmv1/products/iam3/OrganizationsPolicyBinding.yaml

```diff
@@ -0,0 +1,194 @@
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+custom_code:
+  post_delete: 'templates/terraform/post_delete/sleep.go.tmpl'
+autogen_async: true
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+    path: 'name'
+    wait_ms: 1000
+  result:
+    path: 'response'
+    resource_inside_response: true
+  error:
+    path: 'error'
+    message: 'message'
+examples:
+  - name: 'iam_organizations_policy_binding'
+    min_version: 'beta'
+    primary_resource_id: 'my-org-binding'
+    test_env_vars:
+      org_id: 'ORG_ID'
+    vars:
+      pab_policy_id: 'my-pab-policy'
+      display_name: 'test org binding'
+      org_binding_id: 'test-org-binding'
+parameters:
+  - name: 'organization'
+    type: String
+    description: |
+      The parent organization of the Policy Binding.
+    url_param_only: true
+    required: true
+    immutable: true
+  - name: 'location'
+    type: String
+    description: |
+      The location of the Policy Binding
+    url_param_only: true
+    required: true
+    immutable: true
+  - name: 'policyBindingId'
+    type: String
+    description: |
+      The Policy Binding ID.
+    url_param_only: true
+    required: true
+    immutable: true
+properties:
+  - name: 'name'
+    type: String
+    description: |
+      The name of the policy binding in the format `{binding_parent/locations/{location}/policyBindings/{policy_binding_id}`
+    output: true
+  - name: 'uid'
+    type: String
+    description: |
+      Output only. The globally unique ID of the policy binding. Assigned when the policy binding is created.
+    output: true
+  - name: 'etag'
+    type: String
+    description: |
+      Optional. The etag for the policy binding. If this is provided on update, it must match the server's etag.
+    output: true
+  - name: 'displayName'
+    type: String
+    description: |
+      Optional. The description of the policy binding. Must be less than or equal to 63 characters.
+  - name: 'annotations'
+    type: KeyValueAnnotations
+    description: |
+      Optional. User defined annotations. See https://google.aip.dev/148#annotations for more details such as format and size limitations
+  - name: 'target'
+    type: NestedObject
+    description: |
+      Target is the full resource name of the resource to which the policy will be bound. Immutable once set.
+    required: true
+    properties:
+      - name: 'principalSet'
+        type: String
+        description: |
+          Required. Immutable. The resource name of the policy to be bound.
+          The binding parent and policy must belong to the same Organization (or Project).
+        immutable: true
+      - name: 'resource'
+        type: String
+        description: |
+          Output only. The globally unique ID of the policy to be bound.
+        immutable: true
+        output: true
```

Comment:
👋🏻  Hi again!

Your PR generally looks good, I just have one question

[I don't see `resource` as a field that can be set in `target`](https://cloud.google.com/iam/docs/reference/rest/v3beta/folders.locations.policyBindings#PolicyBinding.Target) and the acceptance tests only set target.principal_set, never target.resource. If the field does exist (I assume the public docs are just a bit out of date?) could you please add an acceptance test that uses that field?

Reply:
it does exist but it's hidden for future features. i'll remove it.